nixos/modules/services/web-apps/mastodon.nix

```
@@ -141,6 +141,22 @@ let
     })
   ) cfg.sidekiqProcesses;
 
+  nginxCommonHeaders = ''
+    add_header Strict-Transport-Security 'max-age=63072000; includeSubDomains';
+  ''
+    # Mastodon in upstream is not supported HTTP3 protocol.
```

```suggestion
    # Upstream does not supported HTTP3 protocol
```

Why do we have this, if upstream is not supporting it? Wouldn't we break things then?